src/modules/rust.rs

```diff
@@ -109,6 +111,17 @@ fn execute_rustup_override_list(cwd: &Path) -> Option<String> {
     extract_toolchain_from_rustup_override_list(&stdout, cwd)
 }
 
+fn execute_rustup_default() -> Option<String> {
+    let Output { stdout, .. } = create_command("rustup")
+        .ok()?
+        .args(&["default"])
+        .output()
+        .ok()?;
+    let stdout = String::from_utf8(stdout).ok()?;
+
+    stdout.split_whitespace().next().map(str::to_owned)
+}
```

Comment:
Not a blocker but if possible can you use the `exec_cmd` here since it will also provide some additional logging and automatically kill the command if it takes too long e.g

```suggestion
fn execute_rustup_default(context: &Context) -> Option<String> {
    context
        .exec_cmd("rustup", &["default"])?
        .stdout
        .split_whitespace()
        .next()
        .map(str::to_owned)
}
```